piker/ui/_interaction.py

```diff
@@ -188,7 +188,7 @@ def set_pos(
 
         self._abs_top_right = label_anchor
         self._label_proxy.setPos(self.vb.mapFromView(label_anchor))
-        self._label.show()
+        # self._label.show()
```

Comment:
Ahh this is a me thing that leaked in.
Just trying to rethink the selection UX.

Reply:
#129 related.